new-frontend/src/app/services/permission.service.ts

```diff
@@ -6,6 +6,14 @@ import { BaseUser } from '../models/api/user.model';
 import { ApiService } from './api.service';
 import { USER_ID } from '../models/constants/sessionStorage';
 
+const requiredScopeLevel: Record<ScopeType, ScopeType[]> = {
+  [ScopeType.OWN]: [ScopeType.OWN, ScopeType.ORGANIZATION, ScopeType.COLLABORATION, ScopeType.GLOBAL],
+  [ScopeType.ORGANIZATION]: [ScopeType.ORGANIZATION, ScopeType.COLLABORATION, ScopeType.GLOBAL],
+  [ScopeType.COLLABORATION]: [ScopeType.COLLABORATION, ScopeType.GLOBAL],
+  [ScopeType.GLOBAL]: [ScopeType.GLOBAL],
+  [ScopeType.ANY]: []
```

Comment:
I think ANY scope should be the same as line 10: if you are checking to see if you can do X for any scope, you still need to have permission for *at least 1* scope. Now, I think the check below (line 83-88) will always yield true if you request any scope, while you still need a permission

Reply:
ScopeType.ANY is not used in this PR. It is in the list to prevent Typescript errors. But perhaps better to define it as you suggest.